classes/feature-elementor-show-back-to-wp-editor-button.php

```diff
@@ -0,0 +1,61 @@
+<?php
```

Comment:
@lukechinworth Can you add an update to the readme.txt?

It seems like hiding the "Back to editor" button is starting to make this plugin into more of a "general Elementor utility plugin" vs a dynamic tags focused plugin.

Reply:
done